phpunit.xml.dist

```diff
@@ -12,5 +12,26 @@
 
     <php>
         <ini name="date.timezone" value="UTC"/>
+		<const name="TESTS_ZEND_LDAP_HOST" value="localhost"/>
```

Comment:
folks, what's the valid approach for defining consts? `phpunit.xml` or `tests/TestConfiguration.ci.php`?
I would move them into `TestConfiguration` php file to keep consistency and customization that can be done via copying `tests/TestConfiguration.ci.php` to `tests/TestConfiguration.php`.

Reply:
I guess we can move both TESTS_ZEND_LDAP_HOST & TESTS_ZEND_LDAP_PORT, or just the host. We will see what the host will be for LDAP.

Reply:
https://github.com/zf1s/zf1/pull/159#discussion_r1043251113